### PR TITLE
speed testing added

### PR DESCRIPTION
**Pull request by baentsch**
Closes out #181 

Successful CCI run at https://circleci.com/gh/baentsch/openssl/tree/mb-speedtesting 

(@xvzcf please note spurious connection test errors -- something's still fishy there...) 

##### Checklist
- [x] tests are added or updated

### CODE REVIEW

**Comment by xvzcf** [comment]
The failures are strange indeed. I'm going to try and increase the wait time (before checking the port the server is bound to) to 5 seconds.

If that doesn't fix it, I'll have to figure out a way to wait until the process has finished binding to a port ...

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ localCheckout: &localCheckout
     - run:
         name: Run OQS connection and CMS tests
```
Comment (xvzcf) [inline .circleci/config.yml]: Could you rename this step to "Run OQS connection, speed, and CMS tests"?

File: .circleci/config.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ localCheckout: &localCheckout
             command: .circleci/git_no_checkin_in_last_day.sh || (./Configure ${OPENSSL_LIBTYPE} darwin64-x86_64-cc no-tests && make -j3)
         - run:
             name: Run connection and CMS tests
```
Comment (xvzcf) [inline .circleci/config.yml]: Same here, to "Run OQS connection, speed, and CMS tests".

Comment (baentsch) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 1 above)
Comment (baentsch) [reply .circleci/config.yml]: done

**Review by xvzcf — APPROVED** [review]

**Pull request merged**